commons/src/main/scala/com/virtuslab/vss/common/BaseEndpoints.scala

```diff
@@ -0,0 +1,37 @@
+package com.virtuslab.vss.common
+
+import sttp.tapir.*
+import sttp.tapir.generic.auto.*
+import sttp.tapir.json.upickle.*
+import upickle.default.*
+
+object BaseEndpoints:
+
+  val checkPasswordEndpoint: Endpoint[Unit, CheckPwned, Unit, CheckedPwned, Any] = sttp.tapir.endpoint.post
```

Comment:
[minor] you can use `PublicEndpoint` if there's no secruity inputs, plus is the `sttp.tapir.endpoint` (prefix) intentional? there's a `*` import